.swiftlint.yml

```diff
@@ -1,4 +1,6 @@
 excluded:
+  - Carthage
+  - Pods
   - third_party
 
 disabled_rules:
```

Comment:
Sure, let me remove these — I think we will want to include at least `force_try` and `force_cast` to begin with, since those are harder to remove. Maybe turn them from errors into warnings?